project/BRANCHES-AND-TAGS.md

```diff
@@ -0,0 +1,35 @@
+Branches and tags
+=================
+
+Note: details of the release process for the Engine are documented in the
+[RELEASE-CHECKLIST](https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md).
+
+# Branches
+
+Docker projects should normally have only three living branches at all time, including the regular
+`master` branch:
+
+## `docs` branch
```

Comment:
I know this is the current process, but is there any reason to have a separate docs branch from the release branch? We should publish docs from release/2.0, release/2.1, etc.